spark/src/test/scala/org/apache/spark/sql/delta/rowid/RowIdSuite.scala

```diff
@@ -193,6 +193,23 @@ class RowIdSuite extends QueryTest
     }
   }
 
+  test("Check missing High Watermark") {
```

Comment:
Can we give this a more descriptive name? Right now it's not clear what it checks.